src/theme/components/IconButton.ts

```diff
@@ -0,0 +1,13 @@
+export const IconButton = {
+  BaseStyle: {
+    background: 'tomato',
+  },
+  colorScheme: 'green',
+
+  defaultProps: {
+    colorScheme: 'purple',
+  },
+
+  // DefaultProps: {
+  // },
```

Comment:
Pls remove

Reply:
Actually, the theme for IconButton isn't working. IconButton inherit theme of Button, and can't be styled independently.